syntax = "proto3";
package system_map;




import "easy_work_service_sdk/model/easy_work_service/app_system.proto";

/*
GetCollection返回
*/
message GetCollectionResponse {
    /*
    我收藏的应用系统
    */
    repeated easy_work_service.AppSystem APP_SYSTEM = 1 ;
    /*
    个人收藏 ID
    */
    string instanceId = 2 ;
    /*
    名称
    */
    string name = 3 ;
    /*
    收藏用户名
    */
    string username = 4 ;
}

/*
GetCollectionApi返回
*/
message GetCollectionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    system_map.GetCollectionResponse data = 4 ;
}

